### ISSUE TITLE
Assertion failed: 8 == NN_USOCK_STATE_ACTIVE

### ISSUE THREAD

**Issue by JackDunaway** [nanomsg/nanomsg#645]
Seen in Travis CI build on `ipc_shutdown` test: https://travis-ci.org/nanomsg/nanomsg/jobs/124323052#L666

**Comment by JackDunaway** [comment]
From that log:

```
Assertion failed: 8 == NN_USOCK_STATE_ACTIVE (../src/aio/usock_posix.inc:461)
/bin/bash: line 5: 13806 Aborted                 (core dumped) ${dir}$tst
FAIL: tests/ipc_shutdown
```

**Comment by gdamore** [comment]
I think this is another aspect of #603  -- and fixed by the same bug fix (testing pending).  (This is just on the recv side instead of the send side.)

**Comment by gdamore** [comment]
I'm marking this fixed as a dup of #603 for now.  If we see the problem reoccur after today in Travis, please reopen.
